projects/ng-polymorpheus/src/types/content.ts

```diff
@@ -2,14 +2,16 @@ import {TemplateRef} from '@angular/core';
 import {PolymorpheusComponent} from '../classes/component';
 import {PolymorpheusTemplate} from '../directives/template';
 import {PolymorpheusHandler} from './handler';
-import {PolymorpheusPrimitive} from './primitive';
 
 /**
  * All content types supported by {@link PolymorpheusOutletDirective}
  */
-export type PolymorpheusContent<C extends object = {}> =
+export type PolymorpheusContent<C extends Record<any, any> = {}> =
     | TemplateRef<C>
     | PolymorpheusTemplate<C>
-    | PolymorpheusComponent<object, C>
+    | PolymorpheusComponent<Record<any, any>, C>
```

Comment:
Probably `any`, I think `unknown` might not work.

Reply:
fixed